solr/solr-ref-guide/src/meta-docs/publish.adoc

```diff
@@ -37,6 +37,8 @@ NOTE: Builds are available via https://builds.apache.org/view/L/view/Lucene/[Jen
 
 == Build the DRAFT Guide
 
+NOTE: After the switch to website in GIT, built by Pelican (https://github.com/apache/lucene-site) we no longer need to update `extpaths.txt`. When the RM changes the `LUCENE_LATEST_RELEASE` variable in https://github.com/apache/lucene-site/blob/master/pelicanconf.py in the website git repo, the default redirect for https://lucene.apache.org/solr/guide/mypage.html will automatically redirect to the latest `X_Y` version. Thus you should always publish a DRAFT guide to that location *before* the release of Solr. Then you can publish a final version later, overwriting the previous.
```

Comment:
This paragraph sort of conflates a couple points together IMO.

First, I don't know the value of keeping the idea that there used to be a need to edit the extpaths.txt file actually provides much value - it assumes the reader had a knowledge of how things used to be done, and we know from experience that next to no one does. 

Next, the 2nd sentence assumes a knowledge of the binary release process and only links to a script to provide more context. Perhaps it would just be simpler for these two sentences to just say: 

"The binary release process <<link>> will update the `LUCENE_LATEST_RELEASE` property which will ensure that Ref Guide URLs without a version in the path will automatically redirect to the latest Guide."

And with that simplification, I'm no longer sure warrants a special NOTE callout style.

Lastly, extpaths.txt being gone and the RM changing a parameter are not the reasons why we push a DRAFT version, so the 2nd-to-last sentence implies that it does inappropriately as far as I understand (unless there is a new dependency that's not reflected here that I'm not aware of?). We push the DRAFT so people can look at it as part of the vote process. That may not be appropriately reflected in the rest of the doc, though, so easy to see how it could be confusing.

Reply:
Agree. I added a similar text at the bottom of the page.